lib/pymedphys/_dicom/coords.py

```diff
@@ -132,8 +132,8 @@ def xyz_axes_from_dataset(
     di = float(ds.PixelSpacing[0])
     dj = float(ds.PixelSpacing[1])
```

Comment:
Let's rename di and dj like the following:

```suggestion
    column_spacing = float(ds.PixelSpacing[1])
```

Reply:
@SimonBiggs I uploaded the rtdose file in a new PR. I wait for the approval of the PR before doing the rest in this PR?

Cédric